SPECS/kernel/kernel.spec

```diff
@@ -425,6 +430,11 @@ ln -sf linux-%{uname_r}.cfg /boot/mariner.cfg
 %{_libdir}/perf/include/bpf/*
 
 %changelog
+* Tue Dec 15 2020 Henry Beberman <henry.beberman@microsoft.com> - 5.4.83-1
+- Update source to 5.4.83
+- Address CVE-2020-14351, CVE-2020-14381, CVE-2020-25656, CVE-2020-25704,
+  CVE-2020-29534, CVE-2020-29660, CVE-2020-29661
```

Comment:
`CVE-2020-29660` and `CVE-2020-29661` don't have a corresponding .nopatch file. Is that intended? I think we need it for the tooling to stop marking the package as still having the vulnerability..

Reply:
Thanks for pointing this out. Added in nopatches for those two CVEs as well since it doesn't look like we can count on CPE matching for them.